metrics/configs/flakes-daily-config.yaml

```diff
@@ -0,0 +1,109 @@
+metric: flakes-daily
+query: |
+  #standardSQL
+  select
+    job,
+    consistent_builds,
```

Comment:
Yes that is what the value is counting. This isn't very useful on its own but when influx graphs consider a time range with results from multiple days, they can calculate the chance of an individual build flaking as `1 - sum(consistent_builds)/sum(builds)`. I think that the chance an individual run encounters a flake is an interesting metric especially for ci jobs which may only have 1 commit in a day, but may run many times. If a job like this flakes, it will have a commit consistency of 0, but the build consistency ratio will yield a more accurate flake rate.

Reply:
Why not just use a stability metric? (passed builds)

Reply:
Because if a commit actually breaks a test the job should consistently fail for that commit. This metric will show the builds as not flaking. A stability metric could be useful, especially for CI jobs, but its a different metric than the chance that an individual run of a job fails on a commit that is known to be valid (passed at least once).